app/views/addresses/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= form_with model: [ Current.user, Current.user.addresses.build ] do |form| %>
+  <p>
+    <%= form.label :area %><br>
```

Comment:
@03mayank, create a partial for address form that contains all the common functionalities. And use the partial in new and edit view.